.github/workflows/release-please.yml

```diff
@@ -38,14 +38,24 @@ jobs:
       - name: Build app
         if: ${{ steps.release.outputs.release_created }}
         run: |
-          npm run build
-          npm run server:build
+          npm run build:standalone-app
 
-      - name: Copy generated files to dist
+      - name: Clone the detailed impacts
+        run: |
+          eval `ssh-agent -s`
+          ssh-add - <<< '${{ secrets.PRIVATE_SSH_KEY }}'
+          git clone git@github.com:MTES-MCT/ecobalyse-private.git
+
+      - name: Encrypt the impacts files
+        env:
+          ENCRYPTION_KEY: ${{ secrets.ENCRYPTION_KEY }}
+        run : |
+          npm run encrypt ./ecobalyse-private/data/textile/processes_impacts.json dist/processes_impacts_textile.json.enc
+          npm run encrypt ./ecobalyse-private/data/food/processes_impacts.json dist/processes_impacts_food.json.enc
```

Comment:
Nit: took me a while to realize these encoded files are bundled in the release so self-hosters can configure the app to use them if they provide the encryption key. Might worth adding a comment :)